uiowa/uids
# Feature: uids_base uiowa.edu design updates

# How to test
- [x] https://github.com/uiowa/uids/issues/162
- [x] https://github.com/uiowa/uids/issues/155

- Go to http://localhost:3000/components/detail/header--default and verify that logo placement is lower within tab and that tab does not overlap as much. 
- Verify that http://localhost:3000/components/detail/header--department-nav-with-off-canvas is not altered. 
- Verify that site name, search and menu have all been lowered slightly to stay within the vertical alignment of the block Iowa logo: http://localhost:3000/components/detail/header--site-name-with-off-canvas. 

- [x] https://github.com/uiowa/uids/issues/154

- Test out http://localhost:3000/components/detail/portraits-block on firefox and other browsers.  Verify that line-height is large enough at all breakpoints so that the type descenders can be seen. 

- [x] https://github.com/uiowa/uids/issues/141
- [x] https://github.com/uiowa/uids/issues/140
- [x] https://github.com/uiowa/uids/issues/139

- Verify that http://localhost:3000/components/detail/stat-block has a larger font size on mobile and that prefix and suffix still look good. 

- [x] https://github.com/uiowa/uids/issues/163

- Add a link to the the footer slogan on http://localhost:3000/components/detail/footer--department and verify that it is no longer blue and underlined. 

- [x] https://github.com/uiowa/uids/issues/166

- Add a lot of text to http://localhost:3000/components/detail/group--default and verify that links stay centered in menu. 

- [x] https://github.com/uiowa/uids/issues/167

- Add `is-active` class to `<a>` tag on http://localhost:3000/components/detail/internal-navigation--default and verify that active link is gray instead of orange.
